modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/AutoFocus.es.js

```diff
@@ -40,7 +40,7 @@ export function AutoFocus({children}) {
 						document.activeElement.querySelector('input') &&
 						document.activeElement.querySelector('input').type ===
 							'hidden') ||
-					containerElement.current.parentNode.className.includes(
+					containerElement?.current.parentNode.className.includes(
```

Comment:
I'm adding this Optional Chaining to avoid runtime errors.

Reply:
Also, what I'm trying to do here is consider that a page can contain more than one item in addition to the form. So I'm making the page go through that part of Autofocus and focus directly on the form I'm handling.